src/screens/enter-password/use-enter-password.styles.tsx

```diff
@@ -0,0 +1,13 @@
+import { StyleSheet } from 'react-native';
+
+import { zIndexEnum } from '../../enums/z-index.enum';
+import { useCreateStyles } from '../../styles/use-create-styles';
+
+export const useEnterPasswordStyles = () =>
+  useCreateStyles(({ colors }) => ({
```

Comment:
Omit fn currying, then it will be easier to use these styles inside the component.  
```
export const useEnterPasswordStyles = createStyles(({ colors }) => ...)
...
// inside component
const EnterPasswordStyles = useEnterPasswordStyles();
```